.github/workflows/continuous-integration.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [20.x]
```

Comment:
Are there any doc(like the requirements for deployment the lib) we need to update for node version upgrade?